### PR TITLE
New translations for steplist and datepicker, as well as some fixes

### PR DESCRIPTION
**Pull request by rgeraghty**
- New translations for steplist and datepicker
- some linguistic bug fixes

### CODE REVIEW

File: packages/@react-aria/dnd/intl/sr-SP.json [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+  "dropOnRoot": "Otpusti na",
+  "endDragKeyboard": "Prevlačenje u toku. Pritisnite Enter da biste otkazali prevlačenje.",
+  "endDragTouch": "Prevlačenje u toku. Dvaput dodirnite da biste otkazali prevlačenje.",
+  "endDragVirtual": "Prevlačenje u toku. Kliknite da biste otkazali prevlačenje.",
+  "insertAfter": "Umetnite posle {itemText}",
   "insertBefore": "Umetnite ispred {itemText}",
-  "insertBetween": "Umetnite između {beforeItemText} i {afterItemText}",
-  "insertAfter": "Umetnite posle {itemText}"
+  "insertBetween": "Umetnite između {beforeItemText} i {afterItemText}"
 }
```
Comment (rgeraghty) [inline packages/@react-aria/dnd/intl/sr-SP.json:29]: Our updated automation is sorting this file, result is expected

File: packages/@react-aria/numberfield/intl/sr-SP.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "decrease": "Decrease {fieldLabel}",
-  "increase": "Increase {fieldLabel}",
-  "numberField": "Number field"
+  "decrease": "Smanji {fieldLabel}",
+  "increase": "Povećaj {fieldLabel}",
+  "numberField": "Polje broja"
 }
```
Comment (rgeraghty) [inline packages/@react-aria/numberfield/intl/sr-SP.json:5]: Our updated automation caught this one still in English

File: packages/@react-spectrum/actionbar/intl/cs-CZ.json [hunk 3]
```diff
@@ -2,6 +2,6 @@
   "actions": "Akce",
   "actionsAvailable": "Dostupné akce.",
   "clearSelection": "Vymazat výběr",
-  "selected": "{count, plural, =0 {Nic není vybráno} other {Vybráno #}}",
+  "selected": "Vybráno: { count }",
```
Comment (dannify) [inline packages/@react-spectrum/actionbar/intl/cs-CZ.json:5]: Just double checking we are ok removing the plural specifics here?

Comment (rgeraghty) [reply packages/@react-spectrum/actionbar/intl/cs-CZ.json:5]: This should be okay, as variable still present (count), not all languages work same for plural. This particular string was updated by Leandro as part of his i18n review,

**Review by devongovett — APPROVED** [review]

**Pull request merged**